utils/mkhtml.py

```diff
@@ -305,11 +305,8 @@ def get_addon_path(pgm):
             with open(addons_paths, "r") as f:
                 addons_paths = json.load(f)
             for addon in addons_paths["tree"]:
-                split_path = addon["path"].split("/")
-                root_dir, module_dir = split_path[0], split_path[-1]
-                if "grass8" == root_dir and pgm == module_dir:
-                    return True, addon["path"]
-    return None, None
```

Comment:
I think return None is missing at the end of this function if you keep how this is written. I think you should be returning explicitly every time you use at least one return statement. Maybe Pylint checks for that.